src/questions/derivatives.yaml

```diff
@@ -0,0 +1,951 @@
+- uuid: e0b6dff3-946e-4e8d-9f23-8728fbb37c2a
+  type: SingleChoice
+  content: What primarily determines the value of derivatives?
+  hint: NoHint
+  explanation: Derivatives derive their value from the underlying asset they are representing.
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Demand and Supply of derivative
+      key: A
+    - content: An asset from which it derives its value
+      key: B
+    - content: Fixed Price set by the owner
+      key: C
+    - content: Standardised Prices are set by the Government.
+      key: D
+- uuid: ad7116fc-e76e-4e5b-b98f-55cea5c7e9e3
+  type: SingleChoice
+  content: Which of the following is a financial derivative?
+  hint: NoHint
+  explanation: Financial Derivatives consist of Forwards, Futures, Options, Swaps
+  answerKeys:
+    - C
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Stocks
+      key: A
+    - content: Bonds
+      key: B
+    - content: Futures
+      key: C
+    - content: None of the above 
+      key: D
+- uuid: 16f39eef-f4d9-47dc-9c9d-2f3b34dea7f9
+  type: SingleChoice
+  content: Portfolio Hedging is done by bank manager to- 
```

Comment:
This question needs to be updated as we didn't read about `Reduce interest rate risk.`

All options need to be updated

Reply:
Done